suse_migration_services/units/migrate.py

```diff
@@ -54,6 +55,15 @@ def main():
             ['bash', '-c', bash_command]
         )
     except Exception as issue:
+        root_path = Defaults.get_system_root_path()
+        etc_issue_path = os.sep.join(
+            [root_path, 'etc/issue']
+        )
+        with open(etc_issue_path, 'a+') as issue_file:
```

Comment:
I think we should not append our information but overwrite with open mode ```w```. The reason is this. By default etc/issue has something like "Welcome ..." if we append an error message it sounds like "Welcome ... you are screwed" :-) So I think it's better we just write a clear message and not allow maybe misleading or funny combinations. Thoughts ?

Reply:
Yes, agree that would look weird as well but, I thought what if aother process is using /etc/issue for the same purpose, user could have more info besides "welcome" usual but, yes it should only report about the migration. I will change it.

Reply:
I'm not aware of another process setting /etc/issue. The file belongs to the XXX-release package and I think nothing else modifies it